src/tiled/commandlineparser.cpp

```diff
@@ -72,8 +72,8 @@ bool CommandLineParser::parse(const QStringList &arguments)
         if (arg.length() == 1) {
             // Traditionally a single hyphen means read file from stdin,
             // write file to stdout. This isn't supported right now.
-            qWarning().nospace() << "Bad argument " << index
-                                 << ": lonely hyphen";
+            qWarning().nospace() << tr("Bad argument ") << index
+                                 << tr(": lonely hyphen");
```

Comment:
It seems like you're not aware of how to use string arguments. This should really be:
```c++
qWarning().nospace() << tr("Bad argument %1: lonely hyphen").arg(index);
```
Please also do the "Usage:" line that way with the program name.